src/test/java/com/bean00/datastore/FileSystemDataStoreTest.java

```diff
@@ -10,28 +10,49 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class FileSystemDataStoreTest {
-    private static final String PATH_TO_ROOT = "/Users/jonchin/8th-Light/projects/java-server/cob_spec/public";
+    private static final String PATH_TO_ROOT = "/Users/jonchin/http/public";
```

Comment:
These tests will fail if they run on any computer other than yours.

Reply:
There's still one failing test. You might want to setup CI to avoid these problems. I use Travis CI for all my open source projects. It's pretty easy to setup.

```
-------------------------------------------------------------------------------
Test set: com.bean00.datastore.FileSystemDataStoreTest
-------------------------------------------------------------------------------
Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.348 sec <<< FAILURE! - in com.bean00.datastore.FileSystemDataStoreTest
directoryExists_returnsTrue_whenTheDirectoryExists  Time elapsed: 0.02 sec  <<< FAILURE!
org.opentest4j.AssertionFailedError: 
        at com.bean00.datastore.FileSystemDataStoreTest.directoryExists_returnsTrue_whenTheDirectoryExists(FileSystemDataStoreTest.java:34)
```

Also, you should be aware that working with files in Java is not always this easy. When a project is packaged in a jar, it flattens everything. That means that referencing something by it's path in the source code doesn't work. It this case, it only works because you always have the source code when you are running tests. I usually treat test resources like src resources anyway.

Reply:
Added the directory to make the test pass